keydata.go

```diff
@@ -86,7 +94,7 @@ func (d *keyPolicyUpdateData) Unmarshal(r io.Reader) (nbytes int, err error) {
 	}
 
 	switch version {
-	case 0, 1:
+	case 0:
 		var raw keyPolicyUpdateDataRaw_v0
```

Comment:
this path and other corresponding paths in the file are not tested anymore. If I understand correctly we still need to have this working?